.github/workflows/ci.yml

```
@@ -1,32 +1,11 @@
-name: ci
+name: Verify
 
 on:
   push:
-    branches:
-      - master
   pull_request:
 
 jobs:
-  setup:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Stop old builds
-        if: github.ref != 'refs/heads/master'
-        uses: styfle/cancel-workflow-action@0.9.1
-        with:
-          access_token: ${{ github.token }}
-
   build:
-    strategy:
-      fail-fast: false
-      matrix:
-        java-version: [8,11]
-    needs: setup
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-java@v1
-        with:
-          java-version: ${{ matrix.java-version }}
-      - name: Build & Test
-        run: ./mvnw -B verify
+    name: Verify
+    uses: takari/takari-gh-actions/.github/workflows/ci.yml@v1
+
```

This is almost identically to what we had before, except the newer version of the `action/cehckout` action. As we want to keep Java 8 coverage, I suggest to either drop that change or pass a changed java-version-matrix to that re-used job.

Maybe the following works. If not, the `takari-gh-actions` might need to be changed to support changing that value.

```suggestion
    strategy:
      matrix:
        java-version: ['8', '11', '17']
    name: Verify
    uses: takari/takari-gh-actions/.github/workflows/ci.yml@v1
    with:
      java-version: ${{ matrix.java-version }}
```

See also this docuemntation: https://docs.github.com/en/actions/using-workflows/reusing-workflows#using-a-matrix-strategy-with-a-reusable-workflow